msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -995,6 +995,46 @@ The `extensions.alternates` property is used to hide or prioritize specific in-m
 |`hide.customOfficeAddin`| | | | Configures how to hide an in-market add-in that isn't distributed through AppSource.|
 |`hide.customOfficeAddin.officeAddinId`|String | 64 characters | ✔️ | Specifies the ID of the in-market add-in to hide. The GUID is taken from the app manifest `id` property if the in-market add-in uses the JSON app manifest. The GUID is taken from the `<Id>` element if the in-market add-in uses the XML app manifest. |
 
+## actions
+
+The object is an array of action objects. This block is required only for solutions that provides Actions.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`id`| String | 64 characters | ✔️ | Specifies the ID for the action, which is passed to the code file. |
+|`displayName`| String | 64 characters | | Specifies the display name of the action and it isn't the label of a button or a menu item that invokes the action (which is configured with `tabs.groups.controls.label`).|
+|`description`| String | | ✔️ | Specifies the description of the actions. |
+|`intent`| String enum |  | ✔️ | Specifies the type of actions. The supported enum values are `open`, `addTo`, and `custom`. |
+|`actions.handlers`| Array of objects | | ✔️ | An array of handlers object, defining how Actions can be handled. If an app has more than one handler, only one experience shows up at one entry point. The hub decides which Action to show up based on which experience is supported. |
+
+### actions.handlers
+
+Defines the handlers of the Action. The handlers is an array of handler objects. Each Action must have at least one handler.
+
```

Comment:
Please add the warning section as mentioned in the doc:
https://microsoft-my.sharepoint.com/:w:/p/irenewang/EYpwHSjOT2pAkgn3C4RnoAYBDewN31Z9iMMPi-NOV_HoRw?e=MZSrUF&isSPOFile=1&wdOrigin=TEAMS-MAGLEV.teams_ns.rwc&wdExp=TEAMS-TREATMENT&wdhostclicktime=1702427833680&web=1

Warnings: 

Actions feature is in preview. It is currently only supported in Microsoft 365 app. If you are sideloading to test Actions feature, please sideload using Teams Toolkit in VS code. Upload custom app through Teams UI is not supported yet.